client/components/TestQueueRun/index.jsx

```diff
@@ -66,10 +66,14 @@ class TestQueueRow extends Component {
             // You can only unassign if no tests results have been saved
             if (
                 testers.includes(tester.id) &&
-                this.testsCompletedByUser[tester.id]
+                ! this.testsCompletedByUser[tester.id]
             ) {
                 canUnassignTesters.push(tester);
-            } else if (
+            }
+
+            // You can only unassign if the tester is not assigned alread
```

Comment:
I think this comment should be:  **You can only _assign_ if the tester is not assigned already**